build.gradle

```diff
@@ -64,6 +72,11 @@ dependencies {
   assets files('apache_logo.png')
 }
 
+tasks.withType(JavaCompile) {
```

Comment:
Add a comment above this explaining why this config is here (I can google for what it does, but why it's being added should be captured here so we remember in 6 months).